dangerfile.ts

```diff
@@ -149,10 +149,11 @@ const reactRecogniseProp = ["React does not recognize"]
 const invalidTags = ["The tag <rect>", "The tag <g>", "The tag <text>"]
 const passingTests = ["✓"]
 const unknownEventHandler = ["Unknown event handler"]
-const propType = ["Failed prop type"]
+const propType = ["Failed prop type", "Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.", 'A props object containing a "key" prop is being spread into JSX:']
```

Comment:
The first new addition has been added because we will need to ignore this until we have full conversion of components to Typescript. 

For the second it shall be addressed in this PR or a separate one as this is new